nestkernel/conn_parameter.h

```diff
@@ -69,8 +69,8 @@ class ConnParameter
    * @param rng   random number generator pointer
    * will be ignored except for random parameters.
    */
-  virtual double value_double( index, index, librandom::RngPtr& ) const = 0;
-  virtual long_t value_int( index, index, librandom::RngPtr& ) const = 0;
+  virtual double value_double( thread, librandom::RngPtr& ) const = 0;
+  virtual long_t value_int( thread, librandom::RngPtr& ) const = 0;
```

Comment:
Cyn you please explain why you removed the possibility to use the `sgid` and `tgid` arguments in favor of the `thread` instead of just adding the `thread`? It might be useful to have the former to achieve a better mapping from PyNN and PyNEST.